### PR TITLE
MEN-5096: Add integration test for expiring control maps

### PR DESCRIPTION
**Pull request by oleorhagen**
Changelog: None
Signed-off-by: Ole Petter <ole.orhagen@northern.tech>

### CODE REVIEW

**Comment by oleorhagen** [comment]
Goes along with https://github.com/mendersoftware/mender/pull/910

File: tests/tests/test_update_control.py [hunk 1]
```diff
@@ -193,3 +194,95 @@ def test_update_control_with_broken_map(
...
+    def test_update_control_with_expiring_control_map(
+        self, enterprise_no_client, valid_image,
+    ):
+        """
+        Run an update, in which the download stage takes longer than the
+        expiry time of the control map.
+
+        In other words, test MEN-5096.
+
+        This is done by having an Artifact script pause in Download for a given <tbd> time, then continue on until
```
Comment (lluiscampos) [inline tests/tests/test_update_control.py:207]: tbd? continue on until?

File: tests/tests/test_update_control.py [hunk 2]
```diff
@@ -193,3 +194,95 @@ def test_update_control_with_broken_map(
...
+        deploy = Deployments(auth, devauth)
+
+        devices = list(
+            set([device["id"] for device in devauth.get_devices_status("accepted")])
+        )
+        assert 1 == len(devices)
+
+
+        with tempfile.NamedTemporaryFile(prefix="Download_Leave_01_", mode="w") as sleep_script:
+            # Sleep for x min > UpdateControlMapExpirationTimeSeconds used in the tests
```
Comment (lluiscampos) [inline tests/tests/test_update_control.py:235]:
```suggestion
            # Sleep for 1 min > UpdateControlMapExpirationTimeSeconds used in the tests
```
?

**Review by lluiscampos — APPROVED** [review]
The test looks good! Just couple of nit picks in the comments

File: tests/tests/test_update_control.py (hunk 2 above)
Comment (oleorhagen) [reply tests/tests/test_update_control.py:235]: I'm just removing it all together. Not a good comment.

**Pull request merged**